inst/use_cases/incubation-period.Rmd

```diff
@@ -0,0 +1,117 @@
+
+## Table S2 (Guinea)
+## given a mean and SD with uncertainty around it
+## overall: mean 12.6 (9.1 - 17.1) SD 13.1 (9.6 - 19.6)
+
+## use method of moments to transform mean and SD into useable format
+shape <- function(mean,sd){mean^2/sd^2}
+rate <- function(mean,sd){mean/sd^2}
+
+mean_c <- 12.6
+sd_c <- 13.1
+
+# bounds which we'll use in our simulation to generate the "true" distribution
+mean_l <- 9.1
+mean_u <- 17.1
+sd_l <- 9.6
+sd_u <- 19.6
+
+```
+
+
+```{r,include=FALSE}
+## generate a full set of samples - taken as the "true" distribution which the analysis is based on
+
+n_samples <- 1000
+n_iter <- 100
+
+samples <- matrix(NA,nrow=n_samples,ncol=n_iter)
+for(i in 1:n_iter){
+  
+  ## sample a mean & SD
+  mean_iter <- rnorm(n=1,mean=mean_c,sd=(mean_u-mean_l)/3.92)
+  sd_iter <- rnorm(n=1,mean=sd_c,sd=(sd_u-sd_l)/3.92)
+  
+  # generate parameters accordingly
+  alpha_iter <- shape(mean_iter,sd_iter)
+  beta_iter <- rate(mean_iter,sd_iter)
+  
+  samples[,i] <- rgamma(n=n_samples, shape = alpha_iter, rate = beta_iter)
+}
+
+## for plotting
+samples_longer <- pivot_longer(data.frame(samples),cols=paste0("X",seq(1:n_iter)))
+```
+
+
+
+```{r,include=FALSE} 
+## now we don't have the full set of samples and we try to generate the 95% quantile 
+
+## shape (alpha)
+alpha1 <- shape(mean = mean_c,sd = sd_c)
+
+## rate (beta)
+beta1 <- rate(mean = mean_c,sd = sd_c) 
+
+# estimate 95% quantile
+qgamma(p=0.95,shape = alpha1, rate = beta1) ## 38.08
+
+## alternative method: sample from gamma distribution and take upper quantile
+
+# sample from gamma distribution
+samples1 <- rgamma(n=10000, shape = alpha1, rate = beta1)
+
+# check that things behaving as we would expect
+mean(samples1)
+
+quantile(samples1,0.95) # 38.54
+
+## essentially both are recovering the central estimate well but not anything else
+
+```
+
+The incubation period, defined as the time between infection and the development of symptoms, is an important epidemiological quantity for establishing guidance on the duration of contact tracing of identified cases, as well as the duration of  isolation or quarantine for subsequently identified contacts. It is critical that the duration of isolation mitigates the risk of onward transmission, but does not extend any longer than necessary due to the costs, both financial and social, associated with isolation.\
+The incubation period is most typically presented as a probability distribution, but is often summarised by a mean and estimate of the variability (most commonly standard deviation), rather than the full distribution. \
+Consider the following example. During a novel Ebola outbreak, the incubation period is estimated using a gamma distribution on just a small number of observations (due to the lack of testing and contact tracing). MCMC methods are used to obtain `r n_iter` realisations of a gamma distribution:
+
+```{r,include=TRUE,echo=FALSE}
+ggplot(samples_longer,
```

Comment:
This currently plots to day ~300 on the x-axis, almost all of the density in <100 days so we can shorten the axis.

Reply:
done

Reply:
Thanks!